packages/node_modules/@ciscospark/widget-recents/src/reducer.js

```diff
@@ -3,20 +3,33 @@ import mercury from '@ciscospark/redux-module-mercury';
 import user from '@ciscospark/redux-module-user';
 import spaces from '@ciscospark/redux-module-spaces';
 
-import {UPDATE_STATUS} from './actions';
+import {
+  UPDATE_STATUS,
+  STORE_INCOMING_CALL,
+  DELETE_INCOMING_CALL,
+  UPDATE_WEBRTC_SUPPORT
+} from './actions';
 
 export const initialState = fromJS({
+  incomingCall: null,
```

Comment:
Yea, this doesn't take that into consideration. I think the better way to do this would be to refactor `module-media` to support all of this. There are a couple of other UI elements that weren't designed, like what happens when you load the recents widget and there are active calls that you can join?